packages/ag-charts-website/e2e/api-events.spec.ts

```diff
@@ -165,72 +244,149 @@ test.describe('api-events', () => {
         test.describe('mouse', () => {
             test('nodeClick', async ({ page }) => {
                 await page.mouse.click(noClick.x, noClick.y);
-                await expect(page).toHaveScreenshot('miss-nodeClick.png');
+                expect(consoleLogs).toEqual([]);
 
                 await page.mouse.click(march.x, march.y);
-                await expect(page).toHaveScreenshot('nodeClick-march.png');
+                expect(consoleLogs).toEqual(['[click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n']);
 
                 await page.mouse.click(mayTop.x, mayTop.y);
-                await expect(page).toHaveScreenshot('nodeClick-may-top.png');
+                expect(consoleLogs).toEqual([
+                    '[click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n',
+                    '[click]\nCars sold in May: 42\nNissan: 20\nToyota: 22\n',
+                ]);
 
                 await page.mouse.click(mayBot.x, mayBot.y);
-                await expect(page).toHaveScreenshot('nodeClick-may-bot.png');
+                expect(consoleLogs).toEqual([
+                    '[click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n',
+                    '[click]\nCars sold in May: 42\nNissan: 20\nToyota: 22\n',
+                    '[click]\nCars sold in May: 42\nNissan: 20\nToyota: 22\n',
+                ]);
             });
             test('nodeDoubleClick', async ({ page }) => {
                 await page.mouse.dblclick(noClick.x, noClick.y);
-                await expect(page).toHaveScreenshot('miss-nodeClick.png');
+                expect(consoleLogs).toEqual([]);
 
                 await page.mouse.dblclick(march.x, march.y);
-                await expect(page).toHaveScreenshot('nodeDoubleClick-march.png');
+                expect(consoleLogs).toEqual([
+                    '[click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n',
+                    '[click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n',
+                    '[double click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n',
+                ]);
 
                 await page.mouse.dblclick(mayTop.x, mayTop.y);
-                await expect(page).toHaveScreenshot('nodeDoubleClick-may-top.png');
+                expect(consoleLogs).toEqual([
+                    '[click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n',
+                    '[click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n',
+                    '[double click]\nCars sold in March: 25\nBMW: 10\nToyota: 15\n',
+
+                    '[click]\nCars sold in May: 42\nNissan: 20\nToyota: 22\n',
+                    '[click]\nCars sold in May: 42\nNissan: 20\nToyota: 22\n',
+                    '[double click]\nCars sold in May: 42\nNissan: 20\nToyota: 22\n',
+                ]);
 
                 await page.mouse.dblclick(mayBot.x, mayBot.y);
-                await expect(page).toHaveScreenshot('nodeDoubleClick-may-bot.png');
+                expect(consoleLogs).toEqual([
```

Comment:
Nit: I feel like a more scalable approach would be to clear the logs between groups of assertions here (and in all the other cases of multiple rounds of assertions), so that the assertions are not so dependent on previous test steps?

Reply:
Good idea 👍🏼 Have updated.